release/device-modbus-go.yaml

```
@@ -0,0 +1,19 @@
+---
+name: 'device-modbus-go'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+commitId: '558c40fbdebbe4b00a0283e0ecd9585407ec2bc6'
```

```suggestion
commitId: '3366fcb7b8cfa87a3d941489b4b63a155443012c'
```
This is the final commit for the release.